app/src/main/java/org/simple/clinic/contactpatient/views/RemoveAppointmentReasonItem.kt

```diff
@@ -19,23 +21,25 @@ data class RemoveAppointmentReasonItem(
         reasons: List<RemoveAppointmentReason>,
         selected: RemoveAppointmentReason?
     ): List<RemoveAppointmentReasonItem> {
-      return reasons.map { reason ->
+      return reasons.mapIndexed { index, reason ->
         RemoveAppointmentReasonItem(
             reason = reason,
-            isSelected = reason == selected
+            isSelected = reason == selected,
+            showDivider = index != reasons.lastIndex
         )
       }
     }
   }
 
-  override fun layoutResId(): Int = R.layout.contactpatient_removeappointment_reasonitem
+  override fun layoutResId(): Int = R.layout.removeappointment_reasonitem
 
   override fun render(holder: BindingViewHolder, subject: Subject<Event>) {
-    val binding = holder.binding as ContactpatientRemoveappointmentReasonitemBinding
+    val binding = holder.binding as RemoveappointmentReasonitemBinding
 
     binding.removalReasonButton.setText(reason.displayText)
     binding.removalReasonButton.isChecked = isSelected
     binding.removalReasonButton.setOnClickListener { subject.onNext(Event.Clicked(reason)) }
+    binding.divider.visibleOrGone(showDivider)
```

Comment:
`DividerItemDecorator` was causing UI glitches whenever the list gets updated (the last divider is blinking every time the item selection is changed). To avoid that issue I went with having a divider inside the item.

Reply:
Hmm, this seems alright for now then. But maybe it's something we should revisit and see. A glitching divider is relatively minor compared to the complexity involved in maintaining this, IMO.

Reply:
Yeah, need to revisit the `DividerItemDecorator`. Since that involves playing canvas I didn't want to do it in this PR. Will open a different PR addressing those issues.